### DISCUSSION TITLE
Calendar / Journal - Adding new notes

### DISCUSSION THREAD

**Discussion by Dwindle2234** [TriliumNext/Trilium#3904] [General]
When I save a search I see that the proper 'journal' structure is created.  But how do I create new day notes - from reading the wiki my assumption is that I click a day on a calender and this creates the notes correctly titled but I cannot find this magic calendar.  Am I missing some setup?

![image](https://user-images.githubusercontent.com/120342689/236480049-fd395b5d-1129-4d7a-974f-f173f1b3af5a.png)

**Comment by zadam** [comment]
Hi, you can open the calendar from the left sidebar:

![image](https://user-images.githubusercontent.com/617641/236646545-348624fa-f7e0-4667-98b3-8063ffe4684b.png)

Clicking on a date for which the journal does not exist will create it.

**Comment by Dwindle2234** [comment]
Superb - thank you.  I had removed this from my launchbar.
